sdk/notificationhubs/notification-hubs/README.md

```diff
@@ -655,6 +655,20 @@ console.log(`Notification ID: ${result.notificationId}`);
 
 ## Troubleshooting
 
+## React Native Support
+
+React Native currently does not have support for [`URLSearchParams`] which is used by the Azure Notification Hubs SDK.  In order to use the SDK in React Native, you will need to install the [`url-search-params-polyfill`](https://www.npmjs.com/package/url-search-params-polyfill) package and import it before using the SDK.
+
+```typescript
+import 'url-search-params-polyfill';
+```
+
+This project requires the Web Crypto API to be available in the global scope.  If you are using React Native, you will need to install the [`isomorphic-webcrypto`](https://www.npmjs.com/package/isomorphic-webcrypto) package and import it before using the SDK.
+
+```typescript
+import crypto from 'isomorphic-webcrypto';
```

Comment:
I don't think this would work.  We need the same full list of polyfills in the appconfigBasic RN sample.

Reply:
You're right.  Removing